src/stylesheets/modules/colors.scss

```diff
@@ -41,6 +51,16 @@ $genomix-colors: (
   'brown': $brand-brown,
   'white': $brand-white,
 
+  black: $brand-black,
```

Comment:
to keep it consistent, we should add quotes.  I changed it from earlier where it had no quotes :).  I believe the reason was I was having issues with accessing the color in some cases when it wasn't a string.

Reply:
🙈  my bad. fixed it.

Reply:
not your bad.  I actually had it like this at first too! lol